examples/sample_runtime/src/main.rs

```diff
@@ -119,10 +119,10 @@ impl Runtime for SampleRuntime {
         _snapshot: &dyn Snapshot,
         spec: &InstanceSpec,
         status: InstanceStatus,
-    ) -> Result<(), ExecutionError> {
+    ) {
         match status {
             InstanceStatus::Active => {
-                let instance = self.start_service(spec)?;
+                let instance = self.start_service(spec).unwrap();
```

Comment:
Nit: It could be helpful to add a comment why `unwrap()` is appropriate here, but not when `start_service` is called in `Runtime::initiate_adding_service`.